exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/handlers/RefreshMetadataHandler.java

```diff
@@ -64,6 +65,11 @@ public PhysicalPlan getPlan(SqlNode sqlNode) throws ValidationException, RelConv
       final SchemaPlus schema = findSchema(config.getConverter().getDefaultSchema(),
           refreshTable.getSchemaPath());
 
+      if (schema == null) {
+        return direct(false, "Storage plugin or workspace does not exist [%s]",
```

Comment:
In RefreshMetadataHandler all known exceptions are not thrown but sent as output. Ex:
-----------+
ok	summary
-----------+
false	Storage plugin or workspace does not exist dfs.incorrect
-----------+
The same is done when table doesn't exist or refresh is not supported.
I suggest we use the same / exisitng logic to handle new error.